chain/cmd/request/main.go

```diff
@@ -224,7 +224,7 @@ func main() {
 
 			fmt.Println(tx.SendTransaction(
 				oracle.NewMsgCreateOracleScript(
-					tx.Sender(), "Silly script", "Test oracle script", bytes, tx.Sender()),
+					tx.Sender(), "Silly script", "Test oracle script", bytes, tx.Sender(), "schema", "sourceCodeURL"),
```

Comment:
I think if we don't have schema and source code URL we can set it to an empty string 
@taobun any suggestion